Runtime/src/test/java/com/rakuten/tech/mobile/perf/runtime/internal/UtilSpec.java

```diff
@@ -0,0 +1,120 @@
+import android.content.pm.PackageInfo;
+import android.content.pm.PackageManager;
+import android.content.pm.Signature;
+import com.rakuten.tech.mobile.perf.runtime.RobolectricUnitSpec;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.robolectric.RuntimeEnvironment;
+
+public class UtilSpec extends RobolectricUnitSpec {
+
+  @Mock
+  PackageManager packageManager;
+  Context context;
+  PackageInfo packageInfo = new PackageInfo();
+  final byte[] debugSignatureRaw = {48, -126, 1, -35, 48, -126, 1, 70, 2, 1, 1, 48, 13, 6, 9, 42,
+      -122, 72,
+      -122, -9, 13, 1, 1, 5, 5, 0, 48, 55, 49, 22, 48, 20, 6, 3, 85, 4, 3, 12, 13, 65, 110, 100,
+      114, 111, 105, 100, 32, 68, 101, 98, 117, 103, 49, 16, 48, 14, 6, 3, 85, 4, 10, 12, 7, 65,
+      110, 100, 114, 111, 105, 100, 49, 11, 48, 9, 6, 3, 85, 4, 6, 19, 2, 85, 83, 48, 30, 23, 13,
+      49, 56, 48, 49, 49, 54, 48, 50, 48, 49, 52, 56, 90, 23, 13, 52, 56, 48, 49, 48, 57, 48, 50,
+      48, 49, 52, 56, 90, 48, 55, 49, 22, 48, 20, 6, 3, 85, 4, 3, 12, 13, 65, 110, 100, 114, 111,
+      105, 100, 32, 68, 101, 98, 117, 103, 49, 16, 48, 14, 6, 3, 85, 4, 10, 12, 7, 65, 110, 100,
+      114, 111, 105, 100, 49, 11, 48, 9, 6, 3, 85, 4, 6, 19, 2, 85, 83, 48, -127, -97, 48, 13, 6, 9,
+      42, -122, 72, -122, -9, 13, 1, 1, 1, 5, 0, 3, -127, -115, 0, 48, -127, -119, 2, -127, -127, 0,
+      -101, -90, 82, -72, -124, 12, 63, -33, 117, -49, 75, -68, -85, -51, -89, -71, -58, 0, -17,
+      -115, -85, -37, 72, -17, 96, -101, -59, -7, 87, 27, 93, -78, -98, 20, -52, 64, -40, 42, -90,
+      103, -40, 121, 77, -116, 76, -23, -47, -126, 26, -106, -124, -6, -87, 11, 95, -32, 40, -31,
+      -80, 125, 112, -115, 127, -103, 77, -110, -85, 49, 106, -53, -89, -33, 89, -102, 38, -4, 115,
+      -106, 100, -56, 97, -109, 31, 93, 93, 76, -31, -56, 108, 96, -17, -88, 32, 108, 48, -80, -98,
+      -32, 2, -118, -23, -56, 46, -106, -123, -114, 62, 85, 43, 73, -107, 103, 22, 48, 48, 7, 34,
+      -88, 59, 121, 55, 18, -32, -80, -112, 0, -28, 49, 2, 3, 1, 0, 1, 48, 13, 6, 9, 42, -122, 72,
+      -122, -9, 13, 1, 1, 5, 5, 0, 3, -127, -127, 0, 38, 116, -2, -35, -33, -53, -88, 76, 3, -66,
+      63, 83, -122, -42, 62, -21, -14, -51, 9, 14, 113, -45, -114, -28, -103, -71, -52, 37, -19, 65,
+      -49, 6, -2, 96, 79, -35, 24, -2, -118, 4, 87, 123, 71, -51, -56, -26, 59, -59, -9, -34, -50,
+      -118, 30, -115, -114, -28, -95, -42, -117, 50, 93, 118, 116, -18, -105, -13, -107, -28, 7,
+      -121, -69, -127, -120, -48, -25, -111, -2, -65, -113, 114, 79, -117, -127, -112, 15, 85, -84,
+      12, 31, -50, 110, 88, -105, -103, 111, 117, -109, 13, -126, -77, -112, 68, -24, -48, -45, -81,
+      -76, 116, -37, -85, -80, -97, 59, -92, 93, -52, 6, -97, -69, 108, 90, -74, 79, 91, -91, 57,
+      -102, -85};
+  final Signature[] debugSignatures = {new Signature(debugSignatureRaw)};
+  final byte[] releaseSignatureRaw = {48, -126, 2, -1, 48, -126, 1, -25, -96, 3, 2, 1, 2, 2, 4,
```

Comment:
you could just put this in a resource file and load it in an `@Before` - or is there any reason not to do that?
You could implement a small`org.junit.rules.ExternalResource` similar to `TestData` (or adapt `TestData` to only parse strings on demand)

- [ ] move test data into resource file